src/Canvas.cc

```diff
@@ -768,6 +770,14 @@ NAN_METHOD(Canvas::RegisterFont) {
   g_free(style);
 }
 
+NAN_METHOD(Canvas::DeregisterAllFonts) {
+	// Unload all fonts from pango to free up memory
```

Comment:
Looks like there are some tabs throughout the PR that should be replaced with spaces.

Reply:
All fixed, thanks!